src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Helpers/KnownTypeSymbols.cs

```diff
@@ -49,39 +53,39 @@ internal sealed record KnownTypeSymbols
         public KnownTypeSymbols(CSharpCompilation compilation)
```

Comment:
Consider using a primary constructor so that property declarations and assignments all happen at a single location.

Reply:
I'd do this but some assignments depend on others, e.g. https://github.com/dotnet/runtime/pull/86420/files#diff-14b3ae051a3586abb57cbf4da77639223add78c18359f55b7d66d5449c010f85L65-L66. Might be able to change that later.